scripts/screeps_tencent_batch_rl_runner.py

```
@@ -0,0 +1,1194 @@
+            self.final_status = "failed"
+            self.result.setdefault("error", type(error).__name__ + ": " + str(error))
+            raise
+        finally:
+            self.safe_scale_down()
+            self.finished_at = utc_now_iso()
+            self.write_summary()
+
+    def safe_scale_down(self) -> None:
+        try:
+            self.scale_down()
+        except Exception as error:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)